my-app/src/components/forms/forms.css

```diff
@@ -0,0 +1,93 @@
+.forms{
+    padding-top: -10px;
+}
+  
+.forms h2{
+    padding-top: 10px;
+    padding-left: 500px;
+    font-family: Arial Black;
+    font-weight: 900;
+    font-size: 27px;
+    letter-spacing: 0.1em;
+    color: #413E3E;
+    
+}
+
+.forms h1{
+    padding-top: -100px;
+    font-weight: 300;
+    color: #789954;
+    padding-left: 500px;
+}
+
+.forms h3{
+    word-wrap: break-word;
+    padding-left: 120px;
+    padding-right: 120px;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 100;
+    font-size: 16px;
+}
+
+.forms h4{
+    padding-left: 120px;
+}
+
+.forms .links a, h5 {
```

Comment:
right now when the page is adjusted, the forms heading and links move, so maybe make their position fixed?

Reply:
Also noticed this. The links need to be anchored, especially since on my machine the form links don't appear under the banner, so it looks weird